### ISSUE TITLE
duplicated video section on conference recording in transcode mode when video switches to lower resolution input

### ISSUE THREAD

**Issue by xopnet** [signalwire/freeswitch#763]
When using conference video mode of "transcode" and video switches between a high-resolution stream
and a lower resolution stream, a recording of the full conference shows a duplicated section of the lower resolution stream.

**Comment by mjerris** [comment]
does this happen in mux mode ?

**Comment by xopnet** [comment]
No.  It only happens in transcode mode.  We need to use transcode mode in this case because we only want 
one video stream to be shown at a time (the active-talker).

**Comment by seven1240** [comment]
mp4 is limited to only support one resolution unfortunately. maybe try mkv ?
